kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/CsrServiceImpl.java

```diff
@@ -156,6 +156,11 @@ private boolean isMsisdnInvalid(CallSummaryRecord record) {
     }
 
 
+    private boolean isFirstMessageOfWeek(String weekId) {
+        return weekId.endsWith("1");
```

Comment:
Can you add a comment?  Why does a weekId ending in 1 mean it is the first message of the week?